api/src/schema/orders.ts

```diff
@@ -1,16 +1,10 @@
-import {
-  bigint,
-  numeric,
-  pgTable,
-  serial,
-  timestamp,
-} from "drizzle-orm/pg-core"
+import { bigint, numeric, pgTable, timestamp } from "drizzle-orm/pg-core"
 
 import { people } from "./people"
 import { products } from "./products"
 
 export const orders = pgTable("orders", {
-  id: serial("id").primaryKey().notNull(),
+  id: bigint("id", { mode: "number" }).primaryKey().notNull(),
```

Comment:
nice, I guess this is for consistency with other models which uses `bigint` 👍🏻

Reply:
If IRC on the supabase DB we have `bigint`